content/zh/docs/v3.0/concepts/extensibility.md

```diff
@@ -4,4 +4,59 @@ title: "如何扩展 Dubbo"
 linkTitle: "扩展性"
 weight: 6
 description: "Dubbo 通过 SPI 机制提供了非常灵活的可扩展性"
----
\ No newline at end of file
+---
+
+## 扩展设计理念
+
+可扩展性是任何一个系统所追求的，对于 Dubbo 来说是同样适用。
```

Comment:
I think this part needs to be strengthed, as follows:

+ What's the system extension?
+ What's the benifits of system extension?
+ What are the commonly used extension mechanisms?

Reply:
done